pages/common/tpp.md

```diff
@@ -0,0 +1,11 @@
+# tpp
+
+> Command-Line based presentation tool.
+
+- View a presentation:
+
+`tpp {{filename}}`
+
+- Output a presentation:
+
+`tpp -t {{type}} -o {{outputname}} {{filename}}`
```

Comment:
Perhaps some example extensions would help here. I suspect this could be an example of converting a presentation from a format to another?